anycloud/docs/about.md

```diff
@@ -6,6 +6,6 @@
 
 A high-level overview of how the AnyCloud documentation is organized will help you know how to quickly find what you are looking for:
 
-* The [Tutorial](tutorial.md) will guide you from 0 to a sample Node.js HTTP server deployed in your AWS account with AnyCloud. Start here if you’re new to AnyCloud.
-* [How-to guides](how-to/README.md) are recipes. They guide you through the steps involved in addressing key problems and use-cases. They are more advanced than the tutorial and assume some knowledge of how AnyCloud works.
-* [Technical reference](reference/README.md) for built-in APIs and JSON file configuration schemas. They describe how it works and how to use it but assume some knowledge of how AnyCloud works
\ No newline at end of file
+* The initial [tutorial](tutorials/aws-node.md) will guide you from 0 to a sample Node.js HTTP server deployed in your AWS account with AnyCloud. Tutorials include a [template GitHub repository](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-from-a-template)). Start here if you’re new to AnyCloud.
+* [How-to guides](https://docs.anycloudapp.com/how-to-guides) are recipes. They guide you through the steps involved in addressing key problems and use-cases. They are more advanced than the tutorial and assume some knowledge of how AnyCloud works.
+* [Technical reference](https://docs.anycloudapp.com/technical-reference) for built-in APIs and JSON file configuration schemas. They describe how it works and how to use it but assume some knowledge of how AnyCloud works
```

Comment:
Why do these two need fully-qualified domain names while the others are relative?

Reply:
It is a Gitbook oddity because there is not an explicit README for the section. I'm still looking into that because it opens in a new tab how it is there

Reply:
finally got it. I needed to point to an empty README